### PR TITLE
WRN-1382: Migrate internal/Picker unit tests to Testing Library.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated tests from Enzyme to Testing Library.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed test structure to work with Testing Library.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1382

### CODE REVIEW

File: internal/Picker/tests/Picker-specs.js [hunk 1]
```diff
@@ -284,343 +261,310 @@ describe('Picker Specs', () => {
 	// Note: Because of the limitation of the unit test environment, simulating `keyCode: 13` or the select/enter key will not forward the mouse events that'll trigger `onChange` like it's supposed to
 	test('should increment keyboard select when \'joined\' and \'horizontal\' and wrap successfully', () => {
 		const handleChange = jest.fn();
-		const picker = mount(
-			<Picker index={2} joined max={3} min={0} onChange={handleChange} value={3} />
+		render(
+			<Picker data-testid="picker" index={2} joined max={3} min={0} onChange={handleChange} value={3} />
```
Comment (adrian-cocoara-lgp) [inline internal/Picker/tests/Picker-specs.js:265]: is `data-testid` necessary here?

File: internal/Picker/tests/Picker-specs.js [hunk 2]
```diff
@@ -284,343 +261,310 @@ describe('Picker Specs', () => {
...
-		'should not allow keyboard decrement via down arrow keys when \'joined\' and \'horizontal\'',
-		() => {
-			const handleChange = jest.fn();
-			const picker = mount(
-				<Picker index={0} joined max={1} min={-1} onChange={handleChange} orientation="horizontal" value={0}  />
-			);
+	test('should allow keyboard decrement via up arrow keys when \'joined\' and \'vertical\'', () => {
+		const handleChange = jest.fn();
+		render(
+			<Picker data-testid="picker" index={0} joined max={1} min={-1} onChange={handleChange} orientation="vertical" value={0} />
```
Comment (adrian-cocoara-lgp) [inline internal/Picker/tests/Picker-specs.js:321]: `data-testid` necessary?

File: internal/Picker/tests/Picker-specs.js [hunk 3]
```diff
@@ -284,343 +261,310 @@ describe('Picker Specs', () => {
...
+		expect(actual).toBe(expected);
+	});
 
-			const expected = 0;
-			picker.simulate('keyDown', {keyCode: 38});
-			const actual = handleChange.mock.calls.length;
+	test('should not allow keyboard decrement via left arrow keys when \'joined\' and \'vertical\'', () => {
+		const handleChange = jest.fn();
+		render(
+			<Picker data-testid="picker" index={0} joined max={1} min={-1} onChange={handleChange} orientation="vertical" value={0} />
```
Comment (adrian-cocoara-lgp) [inline internal/Picker/tests/Picker-specs.js:337]: `data-testid` necessary?

File: internal/Picker/tests/Picker-specs.js [hunk 4]
```diff
@@ -284,343 +261,310 @@ describe('Picker Specs', () => {
...
+		fireEvent.keyDown(picker, {keyCode: 39});
+		fireEvent.mouseDown(picker);
+
+		expect(handleChange).not.toHaveBeenCalled();
+	});
+
+	test('should not allow keyboard decrement via down arrow keys when \'joined\' and \'horizontal\'', () => {
+		const handleChange = jest.fn();
+		render(
+			<Picker data-testid="picker" index={0} joined max={1} min={-1} onChange={handleChange} orientation="horizontal" value={0}  />
```
Comment (adrian-cocoara-lgp) [inline internal/Picker/tests/Picker-specs.js:363]: `data-testid` necessary?

File: internal/Picker/tests/Picker-specs.js (hunk 3 above)
Comment (alexandrumorariu) [reply internal/Picker/tests/Picker-specs.js]: Removed it.

File: internal/Picker/tests/Picker-specs.js (hunk 4 above)
Comment (alexandrumorariu) [reply internal/Picker/tests/Picker-specs.js]: Removed it.

File: internal/Picker/tests/Picker-specs.js (hunk 2 above)
Comment (alexandrumorariu) [reply internal/Picker/tests/Picker-specs.js]: Removed it.

File: internal/Picker/tests/Picker-specs.js (hunk 1 above)
Comment (alexandrumorariu) [reply internal/Picker/tests/Picker-specs.js]: Removed it.

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Pull request merged**